crates/bevy_core_pipeline/src/core_2d/mod.rs

```diff
@@ -36,6 +38,7 @@ use bevy_render::{
     Extract, ExtractSchedule, Render, RenderApp, RenderSet,
 };
 use bevy_utils::FloatOrd;
+use nonmax::NonMaxU32;
```

Comment:
Given the prominence of nonmax in bevy_render apis, I think we should probably re-export nonmax from bevy_render so each consumer doesn't need to depend on it.

Reply:
Right, yes. I was planning to do this.

Reply:
Moved to bevy_utils.